content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -248,10 +259,18 @@ Finally replace the serviceaccount with the new updated `sa.yaml` file
 kubectl replace serviceaccount default -f ./sa.yaml
 ```
 
-Now, any new pods created in the current namespace will have this added to their spec:
+### Verify imagePullSecrets was added to pod spec
+
+Now, any new pods created in the current namespace and using default service account will have `spec.imagePullSecrets` field added:
+
+````shell
+kubectl run nginx --image=nginx --restart=Never
+kubectl get pod nginx -o yaml | grep -A 1 imagePullSecrets
```

Comment:
This could also use a [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) query. How about taking that option (rather than `grep`)?

Reply:
Yes, it is much better!